test/units/modules/cloud/misc/conftest.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
This conftest will affect all tests in the same directory. The contents should probably be moved into the test_virt_net.py file instead, unless there is a chance of them being reused with other virt tests. If so, this file and the tests should be placed in a virt subdirectory.

Reply:
I move the files but none of the fixtures of the conftest.py have `autouse=True`. So they won't affect the behavior of the other tests.